generation/core/generator.js

```diff
@@ -163,24 +184,29 @@ module.exports = function({
 	}
 
 	// These types need no wrapping with {type: ..., value: }
-	const plainArgumentTypes = ["id<GREYMatcher>", "String"];
+	const plainArgumentTypes = [
+		"id<GREYAction>",
+		"id<GREYMatcher>",
+		"GREYElementInteraction*",
+		"String"
+	];
 	function shouldBeWrapped({ type }) {
 		return !plainArgumentTypes.includes(type);
 	}
-	function createReturnStatement(classJson, json) {
+	function createReturnStatement(classJson, json, isTranslucent) {
 		const args = json.args.map(
 			arg =>
 				shouldBeWrapped(arg)
 					? t.objectExpression([
-							t.objectProperty(
-								t.identifier("type"),
-								t.stringLiteral(addArgumentTypeSanitizer(arg))
-							),
-							t.objectProperty(
-								t.identifier("value"),
-								addArgumentContentSanitizerCall(arg, json.name)
-							)
-						])
+						t.objectProperty(
+							t.identifier("type"),
+							t.stringLiteral(addArgumentTypeSanitizer(arg))
+						),
+						t.objectProperty(
```

Comment:
Is this formatting the work of prettier ? It's very weird,
<img width="718" alt="screen shot 2018-03-11 at 20 41 43" src="https://user-images.githubusercontent.com/1361404/37257066-a902a83a-256c-11e8-848a-d15215470a46.png">

Reply:
Prettier being prettier 🤷‍♂️